src/plugins/themes.js

```diff
@@ -2,38 +2,35 @@
 
 const fs = require("fs");
 const Helper = require("../helper");
-const colors = require("colors/safe");
 const path = require("path");
 const _ = require("lodash");
 const themes = new Map();
 
 module.exports = {
+	addtheme: addTheme,
```

Comment:
If you call the exported value `addTheme`, then with ES6 this whole thing could be: 

```js
{
	addTheme
  	getAll,
	getFilename,
	loadLocalThemes,
}
```

:)

Reply:
Yeah, I knew about this but I had assumed it didn't work in node 4. Turns out it does, TIL.